Test/test.js

```diff
@@ -1 +1,47 @@
-q
+QUnit.test("addTodo", function(assert){
+  var addTodo = todo.todoFunctions.addTodo;
+  var todos = [];
+  var newTodo = { description: 'td2' };
+
+
+  assert.deepEqual(
+    addTodo(todos, newTodo),
+    [{ id:0, description: 'td2'}],
+    "added the new to do"
+  );
+  assert.deepEqual(
+    addTodo(todos, newTodo),
+    [{ id:1, description: 'td2'}],
+    "added the second to do"
+  );
+
+  assert.deepEqual(
+    addTodo(todos, newTodo),
+    [{id:2, description:'td2'}],
+    "added the third to do"
+  );
```

Comment:
your tests are still doing the same thing. One thing you could do instead is add two new todos and check that they have different id's.